posixlib/src/main/resources/scala-native/aio.c

```diff
@@ -0,0 +1,65 @@
+#include <aio.h>
+#include "aio.h"
+
+// symbolic constants
+
+int scalanative_aio_alldone() { return AIO_ALLDONE; }
+int scalanative_aio_canceled() { return AIO_CANCELED; }
+int scalanative_aio_notcanceled() { return AIO_NOTCANCELED; }
+int scalanative_lio_nop() { return LIO_NOP; }
+int scalanative_lio_nowait() { return LIO_NOWAIT; }
+int scalanative_lio_read() { return LIO_READ; }
+int scalanative_lio_wait() { return LIO_WAIT; }
+int scalanative_lio_write() { return LIO_WRITE; }
+
+// scala native functions
+
+int scalanative_aio_cancel(int fd, struct scalanative_aiocb *sn_aiocb) {
+    return 0;
```

Comment:
As I understand this and following functions are only a stubs. Are they used anywhere? Is it not dangerous to provide functions that be used by user, which are actually not doing anything and don't report they're stubs?    We should either remove them or make them  crash the program.

Reply:
Removed, `aio` code slipped through from a former draft of this PR.